src/lxc/lxccontainer.c

```diff
@@ -3210,9 +3210,9 @@ static struct lxc_container *do_lxcapi_clone(struct lxc_container *c, const char
 
 	// update utsname
 	if (!(flags & LXC_CLONE_KEEPNAME)) {
-		clear_unexp_config_line(c2->lxc_conf, "lxc.utsname", false);
+		clear_unexp_config_line(c2->lxc_conf, "lxc.uts.name", false);
 
-		if (!set_config_item_locked(c2, "lxc.utsname", newname)) {
+		if (!set_config_item_locked(c2, "lxc.uts.name", newname)) {
```

Comment:
Hm, I wonder if we need to support both here since we want to remain backwards compatible up until `LXC 3.0`. So we should probably handle `lxc.utsname` and `lxc.uts.name` here.